src/celeritas/phys/PhysicsData.hh

```diff
@@ -212,6 +222,8 @@ struct PhysicsParamsScalars
     real_type fixed_step_limiter{}; //!< Global charged step size limit [cm]
     bool      enable_fluctuation{}; //!< Enable energy loss fluctuations
 
+    real_type secondary_stack_factor = 3; //!< Secondary storage per state size
```

Comment:
Isn't this redundant as it is also defined in PhysicsParams.hh?

Reply:
The `PhysicsParams::Options` one is the "user input" class: it gets copied over to the `PhysicsParamsScalars` during construction.

Reply:
Whoooops it looks like I'm not actually copying it! 😱  Good catch!